src/bindings/js/node/tests/tensor.test.js

```diff
@@ -1,10 +1,27 @@
-var ov = require('../build/Release/ov_node_addon.node');
+// -*- coding: utf-8 -*-
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+let ov = require('../build/Release/ov_node_addon.node');
 const assert = require('assert');
 const { test, describe, it } = require('node:test');
+const random = require('random-bigint');
```

Comment:
Something meaningful 
```suggestion
const getRandomBigInt = require('random-bigint');
```